packages/paste-core/components/card/src/index.tsx

```diff
@@ -1,25 +1,51 @@
 import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {SpaceProps} from '@twilio-paste/types';
+import {StaticDiv} from '@twilio-paste/types/src/DomTypes';
+import {errorOnBadProps, spacingProps} from './utilities';
 
-interface CardTitleProps {
-  children: React.ReactNode;
-}
-const CardTitle: React.FunctionComponent<CardTitleProps> = ({children}) => <h4>{children}</h4>;
-
-interface CardBodyProps {
-  children: React.ReactNode;
-}
-const CardBody: React.FunctionComponent<CardBodyProps> = ({children}) => <div>{children}</div>;
-
-interface CardProps {
-  onClick: () => void;
-  children: React.ReactNode;
-}
-
-const Card: React.FunctionComponent<CardProps> = ({children, onClick}) => (
-  // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-tabindex
-  <aside onClick={onClick} tabIndex={onClick != null ? 0 : undefined} role={onClick != null ? 'button' : undefined}>
-    {children}
-  </aside>
-);
-
-export {Card, CardBody, CardTitle};
+interface CardFooterProps extends SpaceProps, StaticDiv {}
+
+const cardProps = spacingProps;
+const cardFooterProps = spacingProps;
+
+const CardFooter: React.FunctionComponent<CardFooterProps> = ({children, ...attributes}) => {
+  errorOnBadProps(attributes, cardFooterProps);
+
+  return (
+    <Box
+      borderTopWidth="borderWidth10"
+      borderBottomWidth="borderWidth0"
+      borderLeftWidth="borderWidth0"
+      borderRightWidth="borderWidth0"
+      marginTop="space40"
+      paddingTop="space40"
+      borderColor="colorBorder"
+      borderStyle="solid"
+      {...attributes}
+    >
+      {children}
+    </Box>
+  );
+};
+
+interface CardProps extends SpaceProps, StaticDiv {}
+
+const Card: React.FunctionComponent<CardProps> = ({children, ...attributes}) => {
+  errorOnBadProps(attributes, cardProps);
+
+  return (
+    <Box
+      borderWidth="borderWidth20"
```

Comment:
Still missing setting the `backgroundColor="colorBackgroundBody"`

I'm also tempted to set the `as` prop to be an `<article />`. I'm kind of hoping they are used as distinct pieces of content on the page and articles can provide nice navigation helpers to JAWS users